src/Events/UnknownEvent.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace ZeroDaHero\LaravelWorkflow\Events;
+
+class UnknownEvent extends BaseEvent
```

Comment:
Let's name this `WorkflowEvent` instead. Given the way the dispatcher adapter is connected, this can't come from anywhere else, so it seems safe to just call it a generic "workflow event". (With some of the new changes, we shouldn't ever see this anyway)